docs/versions.rst

```diff
@@ -0,0 +1,7 @@
+Available Documentation for AutoGluon
+-------------------------------------
+
+Web-based documentation is available for versions listed below:
+
+- `AutoGluon 0.0.15 (dev) documentation <https://autogluon.mxnet.io/dev/index.html>`_
+- `AutoGluon 0.0.14 (stable) documentation <https://autogluon.mxnet.io/stable/index.html>`_
```

Comment:
Stable should be the first listed along with a sentence above it that says "Documentation for the latest stable release is available below:", and then below the link there should be a sentence saying that documentation is available for other versions of AutoGluon listed below. This way most users will click the stable link and only advanced users will click the other links.

Reply:
Let's keep this in from most recent to older versions order.